src/Symfony/Component/Validator/Tests/Constraints/UrlValidatorTest.php

```diff
@@ -65,6 +65,30 @@ public function testValidUrls($url)
         $this->assertNoViolation();
     }
 
+    /**
+     * @dataProvider getValidRelativeUrls
+     * @dataProvider getValidUrls
+     */
+    public function testValidRelativeUrl($url)
+    {
+        $constraint = new Url(array(
+            'relativeProtocol' => true,
```

Comment:
should also test absolute urls are still valid

Reply:
The test has both `getValidRelativeUrls` and `getValidUrls` as a data provider. That covers both absolute and relative URL's. Or am I missing something?

Reply:
@MyDigitalLife nice :) didnt noticed that.